src/components/MDX/MDXComponents.tsx

```diff
@@ -30,6 +30,7 @@ import ButtonLink from 'components/ButtonLink';
 import {TocContext} from './TocContext';
 import type {Toc, TocItem} from './TocContext';
 import {TeamMember} from './TeamMember';
+import NextImage from 'next/image';
```

Comment:
use `next/future/image` instead

Reply:
[next/future/image import was renamed to next/image](https://nextjs.org/docs/pages/api-reference/components/image#:~:text=next/future/image,next/image) in v13.0.0. and we are using ^13.4.1. Is it still required?